ontology/peco.md

```diff
@@ -27,6 +27,7 @@ activity_status: active
 repository: https://github.com/Planteome/plant-experimental-conditions-ontology
 preferredPrefix: PECO
 depicted_by: http://planteome.org/sites/default/files/garland_logo.PNG
+added: 2017-06-05
```

Comment:
this isn't correct. This was a prefix move from OE. PECO/EO have been in OBO for over a decade

Reply:
Well let's say that this is indeed the first date that the `peco.md` file was committed to this git repository. I think we can save quite a bit of time if we change the communication about this field rather than the content itself